nox/sessions.py

```diff
@@ -184,6 +185,13 @@ def create_tmp(self) -> str:
         self.env["TMPDIR"] = tmpdir
         return tmpdir
 
+    @property
+    def cache_dir(self) -> pathlib.Path:
+        """Create and return a 'shared cache' directory to be used across sessions."""
+        path = pathlib.Path(self._runner.envdir).parent.joinpath(".shared")
```

Comment:
Just a minor nit here. This could be simplified using `global_config.envdir`, which points to the top-level `.nox` directory:

```suggestion
        path = pathlib.Path(self._runner.global_config.envdir).joinpath(".shared")
```

This avoids some unnecessary path manipulations, and using the parent directory of a virtualenv which does not necessarily exist (`PassthroughEnv` does not create a virtualenv). However, the existing code does work, so this could also be merged as-is IMO.

Reply:
Yeah agree on this 👍🏻

Reply:
SGTM